models/hbert/sentence_encoder.py

```diff
@@ -1,17 +1,15 @@
 from torch import nn
-
-from models.bert.model import BertPreTrainedModel, BertModel
+from transformers import BertPreTrainedModel, BertModel
 
 
 class BertSentenceEncoder(BertPreTrainedModel):
-    def __init__(self, config, num_labels):
-        super().__init__(config)
-        self.num_labels = num_labels
+    def __init__(self, config):
+        super().__init__(config, num_labels=config.num_labels)
         self.bert = BertModel(config)
         self.dropout = nn.Dropout(config.hidden_dropout_prob)
-        self.apply(self.init_bert_weights)
+        self.init_weights()
 
     def forward(self, input_ids, token_type_ids=None, attention_mask=None):
-        _, pooled_output = self.bert(input_ids, token_type_ids, attention_mask, output_all_encoded_layers=False)
+        pooled_output = self.bert(input_ids, token_type_ids, attention_mask)[1]
```

Comment:
They changed the argument positions going from `pytorch-transformers` to `transformers`.

Reply:
Thanks for pointing that out. I fixed this issue in 16dd9d9, along with fixing the weight decay when initializing the AdamW optimizer.